deploy/crds/kieapp.crd.yaml

```diff
@@ -338,6 +338,56 @@ spec:
                   console:
                     description: ConsoleObject configuration of the RHPAM workbench
                     properties:
+                      cors:
```

Comment:
it would be good to add there some description for this object

Reply:
ok